src/modules/feedback/components/__tests__/__snapshots__/FeedbackBox.spec.js.snap

```diff
@@ -39,6 +39,7 @@ exports[`FeedbackBox should match snapshot 1`] = `
     onCommentChanged={[Function]}
   />
   <TextButton
+    disabled={true}
```

Comment:
should I add test cases? e.g. comment not empty or positiveRatingSelected?

Reply:
If you feel like it makes sense and you'd like to do it, sure, gladly.

Reply:
so for now I will leave it as it is. Iwas just a little curious on your testing policy 😁

Reply:
`Uncaught ReferenceError: 'TestingPolicy' is not defined.` :grin: